### DISCUSSION TITLE
forge test -m testPriceFeedVersionIsAccurate isnt working

### DISCUSSION THREAD

**Discussion by Hardeybola** [Cyfrin/foundry-full-course-cu#1472] [General]
Hello guys, i need your help. I've been trying to run this in my foundry, but i've been getting this error

"error: unexpected argument '-m' found"

for context, this is the function i was trying to test

` ` `solidity
function testPriceFeedVersionIsAccurate() public {
        uint256 version = fundMe.getVersion();
        assertEq(version, 4);
    }
}
` ` `

**Comment by uddercover** [comment]
'-m' is now outdated, use '--mt' instead

**Reply by Hardeybola** [reply]
thank you so much
